docs/data/material/varnish/logos/ProjectsLogosDemo.js

```diff
@@ -72,6 +72,48 @@ export default function Demo() {
           <logos.Incubator />{' '}
         </Center>
       </Item>
+
+      <Item>
+        <Center>
+          <ImageListItemBar title="EarthRanger" position="top" />
+          <logos.EarthRanger width="32" height="32" />{' '}
```

Comment:
de we need to set width? we dont in other items on this page

Reply:
the default for the ones coming from AI2 web as-is were too large, so setting it here

Reply:
oic, ya, the fix is different.. ill add a note where it goes